.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: uv python install ${{ matrix.python-version }}
 
       - name: Install dependencies
-        run: uv sync --frozen
+        run: uv sync --python ${{ matrix.python-version }} --frozen
```

Comment:
@zanieb I don't know if this is a bug, but I had to add this because on 3.13 it was not creating a venv with 3.13, but with 3.10. FYI